src/Modules/OnPremisesDataGateway/Commands.OnPremisesDataGateway/help/MicrosoftPowerBIMgmt.OnPremisesDataGateway.md

```diff
@@ -0,0 +1,22 @@
+---
+Module Name: MicrosoftPowerBIMgmt.OnPremisesDataGateway
+Module Guid: 0492de38-4b0e-4c99-b3f3-1ab7e908a955
+Download Help Link: {{Please enter FwLink manually}}
+Help Version: {{Please enter version of help manually (X.X.X.X) format}}
+Locale: en-US
+---
+
+# MicrosoftPowerBIMgmt.OnPremisesDataGateway Module
+## Description
+{{Manually Enter Description Here}}
```

Comment:
Need to fill in this one, the rest in this file will get updated when you fill out the other markdown files. Just make sure to do a build after your last markdown update to ensure this file is up to date